### DISCUSSION TITLE
Migration utility program

### DISCUSSION THREAD

**Discussion by PaulRudin** [v6d-io/v6d#1356] [Q&A]
The FAQ page (question 8) says: "... In this situation, a utility program distributed with vineyard in the initContainer of B2 triggers the migration of O2 from h2 to h3, enabling pod B2 to access O2 locally." Is there any more documentation about this?

**Comment by sighingnow** [comment]
The utility program has been integrated into the vineyard client has is available as the `client.migrate()` API, or `client.get(..., fetch=True)`.

We would update the documentation later.
